scripts/providers/aws/cloud-config.yaml

```diff
@@ -68,7 +68,7 @@ runcmd:
   - ufw --force enable
   - ufw allow 'Nginx Full'
   - ufw allow 'OpenSSH'
-  - curl https://raw.githubusercontent.com/meilisearch/cloud-scripts/v0.22.0/scripts/deploy-meilisearch.sh | bash -s v0.22.0 aws
+  - curl https://raw.githubusercontent.com/meilisearch/cloud-scripts/dump-dir/scripts/deploy-meilisearch.sh | bash -s dump-dir aws
```

Comment:
Not sure to understand why we change the meilisearch version by `dump-dir`?

Reply:
It's for my test ... Sorry I should have put it in draft.